### ISSUE TITLE
Odd indentation within letfn forms.

### ISSUE THREAD

**Issue by dpassen** [weavejester/cljfmt#73]
Presently, `[cljfmt "0.5.3"]` takes the following form (which I consider to be indented correctly):

``` clojure
(letfn [(foo [x]
          (* x x))
        (bar [x]
          (+ (foo x) (foo x)))]
  (bar 5))
```

and suggests/fixes the indentation to:

``` clojure
(letfn [(foo [x]
          (* x x))
        (bar [x]
             (+ (foo x) (foo x)))]
  (bar 5))
```

**Comment by weavejester** [comment]
This does look like a bug! I haven't had time to look into it yet, but it's on my radar.

**Comment by lcodes** [comment]
I just overwrote the indent rule for `letfn` in our project.clj and found the following form to work correctly: `[[:block 1] [:inner 2]]`.

cljfmt no longer reports errors with our letfn forms!

**Comment by JoelSanchez** [comment]
Said workaround works fine...unless you have a `->>` form (and surely others) inside your letfn. It will indent it with 2 spaces (the effect of `:inner 2` I suppose).

**Comment by lcodes** [comment]
Yes I hit that one right now, refactored without a `letfn` for now. But my rule override definitely isn't a fix yet.

**Comment by v-kolesnikov** [comment]
It's really broken.

**Comment by vemv** [comment]
Can we have a release with this fix please?

Thanks - Victor

**Comment by weavejester** [comment]
Released 0.6.4.
